README.md

```diff
@@ -13,7 +13,7 @@ The argocd-vault-plugin works by taking a directory of yaml files that have been
 
 An annotation is used to specify exactly where the plugin should look for the vault values. The annotation needs to be in the format `avp_path: "path/to/vault"`. This is optional and the path prefix can be configured instead (See [Configuration](#configuration))
```

Comment:
can you also update line 14 to change `avp_path: "path/to/vault"` to `avp_path: "path/to/secret"`?